luggage_question_answer.install

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+* Renames Q&A Moderator role to QA Moderator.
+*/
+function luggage_question_answer_update_7000() {
```

Comment:
Okay, just made those changes. I'm also going to change the base branch to development.